### PR TITLE
Reenable chrome tests

### LINKED ISSUE #16135: Re-enable chrome tests
(fix?) and re-enable chrome tests disabled in #16127

### PR DESCRIPTION
**Pull request by seanbudd**
Closes #16135

## Summary by CodeRabbit

- **New Features**
  - Enhanced Chrome command-line arguments to improve testing efficiency and stability .

- **Tests**
  - Re-enabled several test cases related to ARIA role descriptions and navigation in Chrome, improving test coverage.

### CODE REVIEW

**Comment by seanbudd** [comment]
@mltony I've had to disable two of your tests to re-enable chrome testing. Could you look into fixing these up?
The problematic tests are styleNav and textParagraphNavigation

Example failure: https://ci.appveyor.com/project/NVAccess/nvda/builds/50001460/tests

**Comment by mltony** [comment]
@seanbudd, will take a look this weekend.

**Review by michaelDCurran — APPROVED** [review]

File: tests/system/libraries/_chromeArgs.py [hunk 1]
```diff
@@ -14,7 +14,12 @@ def getChromeArgs() -> str:
 	"""
 	return (
 		"chrome"  # Start Chrome
+		" --no-first-run"  # Don't show intro pages for the first run of chrome
 		" --force-renderer-accessibility"  # don't rely on chrome detecting a screen reader.
+		" --ash-no-nudges"  # Prevents Chrome from showing nudge messages.
+		" --browser-test"  # Enable browser test mode, helps reduce flakey tests
```
Comment (michaelDCurran) [inline tests/system/libraries/_chromeArgs.py:20]: Is there anymore info you can provide on how it reduces flaky tests? all in same process, or reduces UI clutter...?

Comment (seanbudd) [reply tests/system/libraries/_chromeArgs.py:20]: There's not much info from chrome here just the following:

>Tells whether the code is running browser tests (this changes the startup URL used by the content shell and also disables features that can make tests flaky [like monitoring of memory pressure])

https://peter.sh/experiments/chromium-command-line-switches/#browser-test

**Pull request merged**